content-api/content-actors/src/main/scala/org/sunbird/content/util/AssetCopyManager.scala

```diff
@@ -17,8 +17,6 @@ import org.sunbird.graph.utils.NodeUtil
 import scala.concurrent.{ExecutionContext, Future}
 
 object AssetCopyManager {
-  implicit val oec: OntologyEngineContext = new OntologyEngineContext
-  implicit val ss: StorageService = new StorageService
```

Comment:
we need StorageService

Reply:
We need OntologyEngineContext as well

Reply:
OntologyEngine is passed with methods, if StorageService is passed with methods this can be removed